src/exampleTypes.js

```diff
@@ -45,20 +46,20 @@ let textOperatorOptions = [
 export default {
   facet: {
     label: 'List',
-    Component: Component(() => (
+    Component: Component(({ node }) => (
       <div>
-        <div>
-          <input type="checkbox" /> Label
-        </div>
-        <div>
-          <input type="checkbox" /> Label
-        </div>
-        <div>
-          <input type="checkbox" /> Label
-        </div>
-        <div>
-          <input type="checkbox" /> Label
-        </div>
+        {_.map(
+          option => (
+            <div key={option.name}>
+              <input
+                type="checkbox"
+                onChange={F.flip(F.lensProp('selected', option))}
+              />{' '}
+              {option.name}
+            </div>
+          ),
+          node.options
```

Comment:
should be `node.context.options` since `context` holds what comes back form the server